src/subtractValue.js

```diff
@@ -0,0 +1,20 @@
+import { flip, subtract } from 'ramda';
+
+/**
+ * Subtracts its first argument from its second argument.
+ *
+ * @func subtractValue
```

Comment:
I'd opt for a name `subtractNum` to be consistent with future `divideNum` naming.